Installation and Docker Chapters

prudhomm opened the pull request:
Documentation for Feel++ with Docker . Basically the chapter on Docker in the book.

it covers 
 - [x] Installation #28 
 - [x] Docker #29

prudhomm committed:
Merge master into feature/docker-framework

prudhomm committed:
Merge master into feature/docker-framework

prudhomm committed:
Creates assets/VMs vs Containers.png

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker-.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker-.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker-.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker-.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feel--docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Creates EN/02-docker/docker-version-mac.txt

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/docker-version-linux.txt

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/docker-version-mac.txt

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/docker-version-mac-bad.txt

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/docker-version-linux-bad.txt

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker-.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm commented:
@gdolle @boileaum I wrote the pages on introducing and installing Docker. Could you review ?
We need to send the link to these pages to the participants of the course friday the latest

I assigned you as reviewers ;)

thx in advance

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feel--docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feel--docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/00-preface/conventions.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/00-preface/conventions.adoc

Auto commit by GitBook Editor

prudhomm commented:
I just added a Conventions section in the Preface of the book

prudhomm committed:
Updates EN/02-docker/feel--docker.adoc

Auto commit by GitBook Editor

prudhomm commented:
I just added a section `First Steps `, please review @gdolle @boileaum

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up fluid model : boundary conditions

gdolle commented on EN/02-docker/what-is-docker-.adoc:
```diff
@@ -0,0 +1,18 @@
+Introducing Docker and Containers
```
This is an introduction to docker. 
This part should be moved directly in the README.md file of the section.

prudhomm replied:
yes may be

prudhomm replied:
I copied a version in  README but keps the one and rename the section `Introduction` . We will decide which is kept finally

gdolle commented on EN/02-docker/installing-docker.adoc:
```diff
@@ -0,0 +1,85 @@
+
+This section covers briefly the installation of Docker. It should be a relatively simple smooth process to install Docker. 
+
+== Channels
```
Not convinced by this section. It should be shorter and just an introduction sentence like, 
"In this introduction we work only with the "stable" version of docker. (Note there's exist an alpha branch we won't)

gdolle commented on EN/02-docker/installing-docker.adoc:
```diff
@@ -0,0 +1,85 @@
+
+The last possibility is to use link:https://docs.docker.com/engine/installation/binaries/[Docker Binaries] to install Docker. This should be used at the last resort if packages are provided either by your distribution or Docker Inc.
+
+== Tested with Docker 1.12
```
Wrong section level. Also I would remove that section. The version should be mention directly in the introduction when we mention the "stable" version of docker. Maybe it would be better to put a little table

| docker | |
|--- | ---|
| branch | _stable_ |
| version | 1.12 |

gdolle commented on EN/02-docker/installing-docker.adoc:
```diff
@@ -0,0 +1,85 @@
+Installing Docker
```
Hum, that section should be in `Installation/docker(recommended)`.
Users will misunderstand otherwise.

Also I would sum up Linux/Mac/Windows in a table directly. (shorter the better (for simple things))

prudhomm replied:
well, IMO, I think the Installation/Docker(recommended) should point to the Docker section and should be redone.

prudhomm replied:
I renamed the section Installation

gdolle replied:
I've an idea, maybe we should just merge the Installation/Docker(recommended) in this page. And for Installation->Mac/Linux/Windows section we introduce in bold :
**"We recommend you to try Feel++ using Docker (see docker section +link) before going further in Feel++ installation"**.

What do you think ? (That way we don't have kind of phantom section :) )

prudhomm replied:
that's what I want to do. Otherwise it is redundant

gdolle commented on EN/02-docker/installing-docker.adoc:
```diff
@@ -0,0 +1,85 @@
+
+In the latter we shall consider only installing and using the _stable channel_.
+
+== Installing Docker
```
Redondent with first level section naming

prudhomm replied:
fixed

gdolle commented on EN/02-docker/feelpp-docker.adoc:
```diff
@@ -0,0 +1,59 @@
+451f851c9d9f: Pull complete 
+55e763f0d444: Pull complete 
+b112a925308f: Pull complete 
+7e6b3a318b2e: Pull complete 
```
Shorten that part with `[...]`

gdolle commented on EN/02-docker/feelpp-docker.adoc:
```diff
@@ -0,0 +1,59 @@
+Feel++ and Docker
```
rather "Feel++ containers"

prudhomm replied:
done

gdolle reviewed (changes requested):
Some part needs to be rearranged.
- Docker installing should be placed in section 2.1.2 installation/docker(recommended).
 (Or we remove it from installation, as you prefer)
- It would be better to use the docker section just to explain how to use docker (pull, edit, run).
    - "Feel++ container" section ok (push + first run)
    - Section to explain how to mount home directory in    interactive docker, show Feel++ files structures in the container (ls). (PS: This can be done afterwards.)

Wait for feedback (we can see that today IRL 👍 ).

prudhomm commented:
@gdolle I forgot to say that I want a chapter which explains a little bit docker and move/dispatch/redo the content of installation/docker(recommended) to the Docker chapter dans keep basic links in this section

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/README.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/README.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/mac.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/README.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/linux.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/linux.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/linux.adoc

Auto commit by GitBook Editor

prudhomm commented:
@gdolle @Trophime @boileaum @romainhild could you review the Installation chapter please ?
I am done with it. I had to re-write most of it.

@gdolle I added the issue/feature #30 - would be awesome to have that for tips, cautions ...

prudhomm commented:
it is here https://book.feelpp.org/content/v/feature/docker-framework/EN/01-installation/

prudhomm committed:
Updates EN/01-installation/building.adoc

Auto commit by GitBook Editor

prudhomm committed:
Merge master into feature/docker-framework

prudhomm committed:
Updates EN/02-docker/README.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/README.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up fluid model : bc

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/09-annexes/calculus.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/09-annexes/compiling-boost.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/linux.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/01-installation/docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/what-is-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
moved

prudhomm committed:
Updates EN/02-docker/what-is-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/09-annexes/compiling-external-projects.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates EN/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm commented:
@feelpp/solving-pdes a first draft of chapters Installation and Docker is now ready. You can review any time. I am now working on the docker images. You should be able to play with the docker images tomorrow.

gdolle committed:
Fix change PR to prepub + merge from prepub

prudhomm committed:
Creates EN/05-learning/Generic/stokes.adoc

Auto commit by GitBook Editor

prudhomm committed:
Merge origin/feature/docker-framework into feature/docker-framework

prudhomm committed:
rename

prudhomm committed:
Updates en/07-quickref/Integrals/integrate.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates book.json

Auto commit by GitBook Editor

gdolle commented:
@prudhomm If the book still fail, change in the book.json
"template" : "ace"  -> "template" : "default" 
the time I solve the bug. (I'll sleep a bit)

vincentchabannes committed:
up fluid : windkessel

vincentchabannes committed:
up fluid model : bc

Trophime reviewed (commented):
In section 2.1.1 I think that working versions of GCC6 should be specified. My experience is that feelpp compiles with gcc 6.xx but the testsuite and some applications (in hifimagnet) fails depending of the version used. gcc 6.2.0 is working fine for me on Debian/testing whereas gcc 6.2.1 is problematic (no test from testsuite pass!!)

* In the recommended libs hdf5 should be added.
* What about adding med-fichier library?
* At the time of the writing Gmsh default is 2.16

In 2.1.3 to improve readability the command line for installing pre-requistes on Ubuntu are two long. It will be better to like in Debian section use ```/``` to split lines.    

In the same section a note on hdf5/med-fichier may be helpfull for Debian. Maybe this note should appear as a footnote in 2.1.1 section for Salome?

In 2.1.5 I have no preview in gitbook-editor?

In 2.2.3 can we also consider to "deliver" a testsuite component to be able? It will be great to have this kind of "reference" testsuite at least for the team...

```-ti``` should be change to ```-it``` to keep the same notations

Trophime commented:
In 2.1.5 it will be nice also to have the feelpp/configure script displayed in the book at least in appendix.
It gives all information needed for the "recommended" pre-requisites for the interested reader.

vincentchabannes committed:
up fluid : bc

prudhomm committed:
Updates en/02-docker/installing-docker.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm commented:
@Trophime thx. I took into account your feedback

prudhomm committed:
Updates en/00-preface/notations.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/00-preface/notations.adoc

Auto commit by GitBook Editor

Trophime commented on en/01-installation/prerequisites.adoc:
```diff
@@ -48,9 +48,10 @@ Here is a list of libraries that we recommend to use jointly with Feel++.
 |===
 | Library | Minimum Version | Maximum Version  | Notes
 
+| HDF5  | --  | --  | Be careful on Debian/sid a new version of HDF5 breaks MED support
```
HDF5 min=1.8.6 max=1.8.16 for MED support

prudhomm replied:
fixed

prudhomm committed:
Updates en/00-preface/notations.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/01-installation/prerequisites.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/qs_laplacian.cpp

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/qs_laplacian.cpp

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/qs_laplacian.cpp

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/qs_laplacian.cpp

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up solid model

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

gdolle committed:
Fix path

gdolle committed:
Merge branch 'feature/docker-framework' of github.com:feelpp/feelpp-book into feature/docker-framework

prudhomm committed:
Updates styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Merge origin/feature/docker-framework into feature/docker-framework

prudhomm committed:
Updates en/styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Updates en/styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/qs_laplacian.cpp

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

gdolle committed:
Really fix all path pdf work

prudhomm committed:
Updates en/styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Updates en/styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Merge origin/feature/docker-framework into feature/docker-framework

prudhomm committed:
Updates styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up solid model

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates styles/website.css

Auto commit by GitBook Editor

prudhomm committed:
Updates en/styles/website.css

Auto commit by GitBook Editor

vincentchabannes committed:
up solid model : add hyperelasticity expression

prudhomm committed:
Updates en/02-docker/feelpp-containers.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/Generic/laplacian.adoc

Auto commit by GitBook Editor

prudhomm committed:
Creates en/05-learning/FSI/readme.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/05-learning/README.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
up

vincentchabannes committed:
up

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Creates en/05-learning/Generic/LevelSet/readme.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

prudhomm committed:
Updates en/SUMMARY.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
Updates EN/07-quickref/Models/Solid/README.adoc

vincentchabannes committed:
Updates EN/07-quickref/Models/Solid/README.adoc

vincentchabannes committed:
Track 2 files into repository.

- modified EN/07-quickref/Models/Fluid/README.adoc
- modified EN/07-quickref/Models/Solid/README.adoc

Auto commit by GitBook Editor

vincentchabannes committed:
Merge remote-tracking branch 'origin/feature/feelmodels' into feature/docker-framework

vincentchabannes committed:
mv Creating a model in Programming with Feel++

vincentchabannes committed:
Updates en/SUMMARY.adoc

vincentchabannes committed:
mv feelmodels into Modeling with feel++

vincentchabannes committed:
Updates en/07-quickref/Models/Fluid/README.adoc

vincentchabannes committed:
Updates en/07-quickref/Models/Fluid/README.adoc

LANTZT committed:
Generic applications pictures (#32)

* Add example images

* add pictures

* Fix laplacian.adoc

* Try to fix images

* Change pictures colour background

* Really fix images background

* Delete meshfeelpp3d.png

* Delete meshfeelpp2d.png

* Fix link

* Fix old link

* Little adjustments

* Little adjustments 2

LANTZT committed:
Add stokes images (#33)

gdolle committed:
Fix pdf empty math crash

gdolle committed:
rename readme README

gdolle committed:
rename readme README

gdolle committed:
mv models into model dir

gdolle committed:
clean dirs

gdolle committed:
Sort files

Please please avoid cross link in the SUMMARY, respect directory scheme
for sanity (every people don't use gitbook-slow-editor.., moreover there
was toc bugs)

gdolle committed:
Remove redondent Models Readme

- There was the same readme in 4 sections. This clean the toc.

gdolle committed:
Use symlink instead
